WindowsServerDocs/remote/remote-desktop-services/clients/windowsdesktop.md

```diff
@@ -27,7 +27,7 @@ You can use the Remote Desktop client for Windows Desktop to access Windows apps
 
 ## Install the client
 
-Choose the client that matches your version of Windows:
+Choose the client that matches your Windows Architecture. The new Remote Desktop Client (MSRDC) supports Windows 10, Windows 10 IoT Enterprise, and Windows 7 client devices. 
```

Comment:
Remote Desktop client should be lower case 'c' for client.

Reply:
Thanks!